tests/acc_test.go

```diff
@@ -176,6 +176,47 @@ func TestLambdaMemorySizeDiff(t *testing.T) {
 	}}).Equal(t, resourceDiffs)
 }
 
+func TestPartialApply(t *testing.T) {
+	localProviderBinPath := ensureCompiledProvider(t)
+	skipLocalRunsWithoutCreds(t)
+
+	// Module written to support the test.
+	localMod, err := filepath.Abs(filepath.Join("testdata", "programs", "ts", "partial-apply", "local_module"))
+	require.NoError(t, err)
+
+	testProgram := filepath.Join("testdata", "programs", "ts", "partial-apply")
+	localPath := opttest.LocalProviderPath("terraform-module", filepath.Dir(localProviderBinPath))
+	integrationTest := pulumitest.NewPulumiTest(t, testProgram, localPath)
+
+	// Get a prefix for resource names
+	prefix := generateTestResourcePrefix()
+
+	// Set prefix via config
+	integrationTest.SetConfig(t, "prefix", prefix)
+
+	// Generate package
+	pulumiPackageAdd(t, integrationTest, localProviderBinPath, localMod, "localmod")
+	_, err = integrationTest.CurrentStack().Up(integrationTest.Context(), optup.Diff(),
+		optup.ErrorProgressStreams(os.Stderr),
+		optup.ProgressStreams(os.Stdout),
+	)
+	assert.Errorf(t, err, "expected error on up")
```

Comment:
Can we check the state file to see if the resource records for child resources that did succeed got written out?

Can we also see what the module output values are?